src/shared/config.ts

```diff
@@ -34,6 +34,31 @@ export interface ServiceConfig {
   PORT: number;
   HOST: string;
 
+  // ../lib/config.
+  HOSTNAME: string;
```

Comment:
let use HOST instead of HOSTNAME

Reply:
Will be addressed in https://github.com/mojaloop/auth-service/pull/9